hack/make/test-unit

```diff
@@ -1,58 +1,6 @@
 #!/usr/bin/env bash
 set -e
 
-# Run Docker's test suite, including sub-packages, and store their output as a bundle
-# If $TESTFLAGS is set in the environment, it is passed as extra arguments to 'go test'.
-# You can use this to select certain tests to run, e.g.
-#
-#   TESTFLAGS='-test.run ^TestBuild$' ./hack/make.sh test-unit
-#
-bundle_test_unit() {
-	TESTFLAGS+=" -test.timeout=${TIMEOUT}"
-	INCBUILD="-i"
-	count=0
-	for flag in "${BUILDFLAGS[@]}"; do
-		if [ "${flag}" == ${INCBUILD} ]; then
-			unset BUILDFLAGS[${count}]
-			break
-		fi
-		count=$[ ${count} + 1 ]
-	done
+>&2 echo "DEPRECATED: use hack/test/unit instead of hack/make.sh test-unit"
```

Comment:
I think it's confusing to have the redirection in front of the command. In fact I did not even know that this is possible. I'd prefer to have it at the end of the command.